erpnext/accounts/report/accounts_receivable/accounts_receivable.py

```diff
@@ -1146,7 +1150,27 @@ def get_dn_details(party_type, voucher_nos):
 
 	return dn_details
 
-def get_voucher_details(party_type, voucher_nos, dn_details):
+def get_ro_details(party_type, voucher_nos):
+	ro_details = frappe._dict()
+
+	if party_type ==  'Customer':
+		for si in frappe.db.sql("""
+			select
+				parent, GROUP_CONCAT(DISTINCT project SEPARATOR ', ') as ro
+			from
+				`tabSales Invoice Item`
```

Comment:
Do not get Projects from `Sales Invoice Item`, get projects from `Sales Invoice` instead. Do not rely on GROUP_CONTACT either